build/Targets/References.targets

```diff
@@ -222,7 +222,7 @@
       <ItemGroup>
 
         <PackageReference Include="Moq" Version="4.2.1402.2112" />
-        <PackageReference Include="xunit" Version="2.1.0" />
+        <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
```

Comment:
Why do we need 2.2.0 and why a beta4 build and 2.2.0 itself? @davkean [had mentioned](https://github.com/dotnet/project-system/pull/1912#issuecomment-291021172) that 2.2.0 regressed perf when he had tried to upgrade.